arch/Config.pl

```diff
@@ -702,6 +708,17 @@
 	$_ =~ s:CONFIGURE_PNETCDF_LIB_PATH::g ;
 	 }
 
+    if ( $sw_adios2_path ) 
+      { $_ =~ s/CONFIGURE_WRFIO_ADIOS2/wrfio_adios2/g ; 
+	      $_ =~ s:CONFIGURE_ADIOS2_FLAG:-DADIOS2: ;
+        $_ =~ s:CONFIGURE_ADIOS2_LIB_PATH:-L\$\(WRF_SRC_ROOT_DIR\)/external/io_adios2 -lwrfio_adios2 -L$sw_adios2_path/lib -ladios2_fortran_mpi -ladios2_fortran:;
```

Comment:
When I install ADIOS2, the libraries are placed in a `lib64` directory rather than a `lib` directory. I recall on PR #1666 that @fengggli had found that their libraries were installed in a `lib` directory, though. Perhaps there needs to be some additional instruction on how to install ADIOS2, or -- better yet, in my opinion -- we could add some logic to determine where the ADIOS2 libraries are located?

Reply:
@mgduda I installed ADIOS2 2.8.0 on two machines, on my Ubuntu20.04 it's in /lib, but on another HPC system (PSC Bridges2, with CentOS 8.2) it's in /lib64.

@MichaelLaufer  ADIOS2 installation comes with a `adios2-config` tool at `ADIOS2-install-dir/bin/adios2-config`, and it shows the library path correctly for the above two environments. Below is an example on my Ubuntu (perhaps useful...):
```bash
fengggli@DESKTOP-B450FL(:):install/bin/adios2-config -f
-DADIOS2_USE_MPI -I/home/fengggli/Workspace/insituLauncher/install/include/adios2/fortran -Wl,-rpath,/home/fengggli/Workspace/insituLauncher/install/lib /home/fengggli/Workspace/insituLauncher/install/lib/libadios2_fortran_mpi.so.2.8.0 /home/fengggli/Workspace/insituLauncher/install/lib/libadios2_fortran.so.2.8.0 -Wl,-rpath-link,/home/fengggli/Workspace/insituLauncher/install/lib
fengggli@DESKTOP-B450FL(:):install/bin/adios2-config --fortran-flags
-DADIOS2_USE_MPI -I/home/fengggli/Workspace/insituLauncher/install/include/adios2/fortran
fengggli@DESKTOP-B450FL(:):install/bin/adios2-config --fortran-libs
-Wl,-rpath,/home/fengggli/Workspace/insituLauncher/install/lib /home/fengggli/Workspace/insituLauncher/install/lib/libadios2_fortran_mpi.so.2.8.0 /home/fengggli/Workspace/insituLauncher/install/lib/libadios2_fortran.so.2.8.0 -Wl,-rpath-link,/home/fengggli/Workspace/insituLauncher/install/lib
```

Reply:
@mgduda and @fengggli thank you for the valuable feedback. 
I have just pushed a commit that addresses this by adding the `/lib64` path in addition to `/lib` for the ADIOS2 lib path. In my testing this works as expected.

When, and if this makes it to mainline WRF I will update the Spack WRF package that will streamline this process further. A simple `spack install wrf +adios2` is all that will be required. I have also tested this locally and it works smoothly.